packages/vkui/src/components/Footer/Footer.tsx

```diff
@@ -4,14 +4,27 @@ import { HasComponent } from '../../types';
 import { Footnote } from '../Typography/Footnote/Footnote';
 import styles from './Footer.module.css';
 
-export type FooterProps = React.AllHTMLAttributes<HTMLElement> & HasComponent;
+export type FooterProps = React.AllHTMLAttributes<HTMLElement> &
+  HasComponent & {
+    mode?: 'group' | 'list' | 'loader';
+  };
+
+const modeStyles = {
+  group: styles['Footer--mode-group'],
+  list: styles['Footer--mode-list'],
+  loader: styles['Footer--mode-loader'],
+};
 
 /**
  * @see https://vkcom.github.io/VKUI/#/Footer
  */
-export const Footer = ({ children, className, ...restProps }: FooterProps) => {
+export const Footer = ({ children, className, mode = 'list', ...restProps }: FooterProps) => {
   return (
-    <Footnote Component="footer" {...restProps} className={classNames(styles['Footer'], className)}>
+    <Footnote
+      Component="footer"
+      className={classNames(styles['Footer'], modeStyles[mode], className)}
+      {...restProps}
+    >
```

Comment:
О, раз мы тут, то внесу предложение, от которого не стоит отказываться! 😉 

```suggestion
    <Footnote
      Component="footer"
      role="contentinfo"
      className={classNames(styles['Footer'], modeStyles[mode], className)}
      {...restProps}
    >
```
Артем советует для лучшей доступности со скринридера к тегам семантических областей добавлять роли. Тегу `footer` соответствует `role="contentinfo"`.

Reply:
Почитала чутка, с ролью "contentinfo" должен быть только один элемент на странице, групп у нас может быть миллион, вроде бы нужно обязательно тогда добавлять `aria-label`, может, это вынести в `warnOnce`?

Reply:
Зря быканула, резолвлю в пользу отдельного issue: 
- https://github.com/VKCOM/VKUI/issues/4692